src/vulkan/sampler_descriptor.h

```diff
@@ -21,6 +21,8 @@
 namespace amber {
 namespace vulkan {
 
+VkSamplerCreateInfo GetSamplerCreateInfo(Sampler* sampler);
```

Comment:
What about creating a amber::vulkan::Sampler class. It would have a `static VkSamplerCreateInfo GetSamplerCreateInfo(Sampler* sampler)` and then a the init and destructor methods to create and destroy the sampler. Then the image descriptor above would hold a vulkan::Sampler and not need to have the destructor code added?

Reply:
Done